docs/ossm/README.md

```diff
@@ -6,4 +6,80 @@ This documentation is specific to the OpenShift Service Mesh product and may dif
+As the control plane with cluster-wide model watches all namespaces, it's required to restrict that to namespaces only belonging to OSSM 2 mesh. Otherwise it would conflict with OSSM 3. It can be done via [discovery selectors](https://docs.redhat.com/en/documentation/openshift_container_platform/4.17/html/service_mesh/service-mesh-2-x#ossm-excluding-namespaces-from-cluster-wide-mesh-console_ossm-deployment-models) or by switching to multi-tenant deployment model.
+1. configure discovery selectors in your SMCP CR:
+    ```yaml
+    apiVersion: maistra.io/v2
+    kind: ServiceMeshControlPlane
+    metadata:
+      name: basic
+      namespace: istio-system
+    spec:
+      policy:
+        type: Istiod
+      addons:
+        grafana:
+          enabled: false
+        kiali:
+          enabled: true
+        prometheus:
+          enabled: true
+      telemetry:
+        type: Istiod
+      version: v2.6
+      mode: ClusterWide
+      meshConfig:
+        discoverySelectors:
+          - matchExpressions:
+            - key: maistra.io/member-of
+              operator: Exists
+      runtime:
+        components:
+          pilot:
+            container:
+              env:
+                ENABLE_ENHANCED_RESOURCE_SCOPING: 'true'
+    ```
+
+### OSSM 3 installation
+See OSSM [installation](https://docs.openshift.com/service-mesh/3.0.0tp1/install/ossm-installing-openshift-service-mesh.html) documentation for more information.
+1. install `Red Hat OpenShift Service Mesh 3` operator
+1. create `IstioCNI` resource in `istio-cni` namespace
+1. create following `Istio` resource in `istio-system3` (must be a different namespace than a namespace running OSSM 2). Make sure to use discovery selectors which are ignoring OSSM 2 namespaces and NOT to use `default` name for `Istio` resource. You can optionally restrict discovered namespaces even more. Configuration shown in the example only ignores OSSM 2 namespaces but all other namespaces will be part of OSSM 3 mesh. See upstream Istio documentation for [discoverySelectors](https://istio.io/v1.19/docs/setup/install/multiple-controlplanes/) usage.
+    ```yaml
+    kind: Istio
+    apiVersion: sailoperator.io/v1alpha1
+    metadata:
+    name: ossm3
+    spec:
+      namespace: istio-system3
+      values:
+        meshConfig:
+          discoverySelectors:
+            - matchExpressions:
+              - key: maistra.io/member-of
+              operator: DoesNotExist
+      updateStrategy:
+        inactiveRevisionDeletionGracePeriodSeconds: 30
+        type: InPlace
+      version: v1.23.0
+    ```
+1. Deploy your workloads and label the namespaces with `istio.io/rev=ossm3` label
+> **_NOTE:_** Do NOT use `istio-injection=enabled` label for your OSSM 3 workload namespaces unless you have changed `spec.memberSelectors` in `ServiceMeshMemberRoll`.
+
+## Multitenant deployment model
```

Comment:
Suggestion - most users are likely using multi-tenant mode, as it is the default. I would put this first and include (default) in the title. 

I think there are probably many users that didn't think about this at all, just created a mesh using the defaults in the instructions and they don't talk about "multitenant" anywhere.

Reply:
Done, thank you.